lib/src/models/transaction_uploader.dart

```diff
@@ -93,15 +93,11 @@ class TransactionUploader {
 
     if (!_txPosted) {
       await _postTransaction();
-      return;
```

Comment:
What's the rationale for proceeding here? Is it that _postTransaction will now retry until there's been too many failures, and so you can be assured that tx posting is done by the time the function exits?